third_party/libunwindstack/tests/DwarfDebugFrameTest.cpp

```diff
@@ -70,57 +70,73 @@ static void SetCie64(MemoryFake* memory, uint64_t offset, uint64_t length,
   memory->SetMemory(offset, data);
 }
 
-static void SetFde32(MemoryFake* memory, uint64_t offset, uint32_t length, uint64_t cie_offset,
-                     uint32_t pc_start, uint32_t pc_length, uint64_t segment_length = 0,
-                     std::vector<uint8_t>* data = nullptr) {
+template <typename AddressType>
+void SetDataPointerSize(MemoryFake*, uint64_t, AddressType) {}
```

Comment:
Should this contain a static assert as above? Might not be needed since this is just a test.

Reply:
Done.